config.py

```diff
@@ -844,6 +844,10 @@ def create_transaction(db):
     # Feature Flag: Enables notifications about vulnerabilities to be sent for new pushes
     FEATURE_SECURITY_SCANNER_NOTIFY_ON_NEW_INDEX = True
 
+    # Set minimal security level for new notifications on detected vulnerabilities. Avoids
+    # creation of large number of notifications after first index. If not defined, defaults to "High".
+    NOTIFICATION_MIN_SEVERITY_ON_NEW_INDEX = ""
```

Comment:
Can you set it to "High" if that should be the default behavior?

Reply:
@kleesc I set it in `secscan_v4_model.py`. Do I need to put it in the `config.py` as well?

Reply:
Just so it's easier to figure out what the default value is